src/UI/state/mapSlice.ts

```diff
@@ -37,7 +37,7 @@ export interface MapState {
     baseMap: boolean;
   };
 
-  species_list: { species: string }[];
+  species_list: { series: string; color: number[] }[];
```

Comment:
I'm surprised (and slightly concerned) that this change hasn't made a test break in the slice tests... Can you change the test to reflect this change please?